js/unit_tests.ts

```diff
@@ -25,6 +26,14 @@ test(function tests_console_assert() {
   assertEqual(hasThrown, true);
 });
 
+test(function tests_console_stringify_complex_objects() {
+  // tslint:disable:no-any
+  assertEqual("foo", stringify(new Set<any>(), "foo"));
+  assertEqual(`[ "foo", "bar" ]`, stringify(new Set<any>(), ["foo", "bar"]));
+  assertEqual(`{ foo: "bar" }`, stringify(new Set<any>(), { foo: "bar" }));
+  // tslint:enable:no-any
+});
```

Comment:
Cool - Thanks.

Reply:
If you want to do a follow up PR - I would like to have these changes:
1. modifying tests_console_stringify_circular to call stringify instead that would be appreciated.
1. I would also like all the console tests to be moved to `js/console_test.ts` in a similiar way as `compiler_test.ts`
1. Use camel-case instead of snake-case for console test names. Follow `compiler_test.ts` as an example.

Reply:
Alright, I've updated and I'll make another PR with those changes.